modules/reference/pages/schedule/maintenance-windows.adoc

```diff
@@ -0,0 +1,14 @@
+[[schedule-maint-windows]]
+= Maintenance Windows
+
+The menu:Schedule[Maintenance Windows] section allows you to manage your maintenance window schedules.
+To create a maintenance window, you need to have an existing [path]``.ical`` file that contains the calendar.
```

Comment:
```suggestion
To create a maintenance calendar, you need to have an existing [path]``.ical`` file that contains the calendar.
```

Reply:
There are 2 methods of creating a *Calendar*: one is (as you have described) by uploading the iCalendar file. The other is by providing a link (`http`, `https`) and SUMA will download the iCalendar data from it.